Update Instruction for "Updating Angular CLI" if using NPM v5

### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [x] feature request
```

### Versions.
```
cli: 1.4.4
node: 8.6.0
npm: 5.4.2
```


### Repro steps.
```
There is some instruction in Angular CLI , about updating version 

npm uninstall -g @angular/cli
npm cache clean
npm install -g @angular/cli@latest

If you run npm cache clean in npm 5.x , it will give the following error :

npm ERR! As of npm@5, the npm cache self-heals from corruption issues and data extracted from the cache is guaranteed to be valid. If you want to make sure everything is consistent, use 'npm cache verify' instead.
npm ERR! 
npm ERR! If you're sure you want to delete the entire cache, rerun this command with --force.
```


### The log given by the failure.
```
npm ERR! As of npm@5, the npm cache self-heals from corruption issues and data extracted from the cache is guaranteed to be valid. If you want to make sure everything is consistent, use 'npm cache verify' instead.
npm ERR! 
npm ERR! If you're sure you want to delete the entire cache, rerun this command with --force.
```


### Desired functionality.
`Change **npm cache clean** step to **npm cache verify**`

### Mention any other details that might be useful.
```
Use NodeJS version 8.6.0 and NPM v5.4.2 .
NPM v5 no need to clean cache. the npm cache self-heals from corruption issues and data extracted from the cache is guaranteed to be valid.
```

--- a/README.md
+++ b/README.md
@@ -128,6 +128,7 @@ Global package:
 ```bash
 npm uninstall -g @angular/cli
 npm cache clean
+# if npm version is > 5 then use `npm cache verify` to avoid errors (or to avoid using --force)
 npm install -g @angular/cli@latest
 ```
 

